admin/acceptance.py

```diff
@@ -321,6 +321,8 @@ class RunOptions(Options):
          'Version of flocker to install'],
         ['build-server', None, 'http://build.clusterhq.com/',
          'Base URL of build server for package downloads'],
+        ['certificate-directory', None, None,
+         'Directory containing TLS certificates (if using vagrant)'],
```

Comment:
There is no reason to make this configurable. The vagrant box will only ever have particular certificates.